src/pages/storey/containers.mdx

```diff
@@ -43,6 +43,32 @@ container.
 
 To avoid key collisions, you must provide a different byte to each container.
 
+# Accessor
+
+Every container has a corresponding "accessor" type that is used to perform all read/write
+operations.
+
+To get the accessor of a top-level container, the [`access` method] must be called. It takes a reference
+to a storage backend.
+
+```rust template="storage"
+use cw_storey::containers::Item;
+use cw_storey::CwStorage;
+
+let item: Item<u32> = Item::new(0);
+
+item.access(&mut CwStorage(&mut storage))
```

Comment:
Can't this `&mut CwStorage(` conversion be automated somehow?

Reply:
AFAIR it has to do with orphan rules. I didn't initially find a good way to do it, but it's on my mind

Reply:
I am wondering - `CwStorage` is a thin wrapper over a `&mut T` (with some trait bounds on T, irrelevant). `access` function takes a `&mut impl Storage` there (and `CwStorage` implements `Storage`). I can think of two approaches:
1. Types implementing `Storage` should be thin types - if so, then why not simply take this by owned value (like `mut impl Storage`)? Then you don't really need to take a mutable reference here
2. Types implementing `Storage` are not guaranteed to be thin types. In this case, why not to use `CwStorage` here instead? It would still look better - `item.access(&mut CwStorage(storage))`. Often even:
```
let storage = &mut CwStorage(storage);
item.access(storage);
```
Any of those approaches you safe one `&mut`. Also adding another abstraction layer could be nice here - inspired by iterators. Let's create a trait:
```
trait IntoStorage: Sized {
  type Output: Storage;
  
  fn into_storage(self) -> Self::Storage;
}

impl<T: Storage> IntoStorage for T {
  type Output = Self;

  fn into_storage(self) -> Self { self }
}

impl IntoStorage for cosmmwasm_std::Storage {
  type Output = CwStorage;
  
  fn into_storage(self) -> Self::Storage;
}
```

Then you can define `access` more like:

```
fn access(&self, storage: impl IntoStorage) -> ItemAccess<...> {
  let mut storage = storage.into_storage();
  // usual implementation
}
```

I believe to maintain the backward API compatibility you could implement `Storage` and `IntoStorage` also on `&mut CwStorage`/`&mut cosmwams_std::Storage`.

Just an idea how to simplify this API, let's make an issue in Storey from this so we can move the discussion there.

Reply:
At a glance, it sounds promising!